src/pages/about.js

```diff
@@ -29,12 +29,6 @@ module.exports = React.createClass({
         <img className="homepage-image" src="/assets/images/homepage-images/teacher-full.jpg" alt="Man pointing at student's computer screen"/>
       </picture> );
     }
-    if (test === "img-b") {
-      return (<picture>
-        <source srcSet="/assets/images/homepage-images/believe-wide.jpg" media="(max-width: 799px)" />
```

Comment:
Should we remove the 2 image files from repo or is it better that we leave them there in case future A/B testing needs it?

Reply:
Good call. Done. Let's hope Travis doesn't get upset.